src/api/query-hooks/index.ts

```diff
@@ -23,14 +23,36 @@ import {
   getTopologyComponentLabels,
   getTopologyComponents
 } from "../services/topology";
-import { getPersons } from "../services/users";
+import { getPersons, getVersionInfo } from "../services/users";
+import packageJson from "../../../package.json";
 
 const cache: Record<string, any> = {};
 
 const defaultStaleTime = 1000 * 60 * 5;
 
 export const createIncidentQueryKey = (id: string) => ["getIncident", id];
 
+export const useVersionInfo = () => {
+  return useQuery(
+    ["version-info"],
+    () => {
+      return getVersionInfo().then((data) => {
+        const versionInfo: any = data.data || {};
+        data.data = {
+          ...versionInfo,
+          frontend: packageJson.version,
+          backend: versionInfo.Version
+        };
+        return data;
+      });
+    },
+    {
+      staleTime: defaultStaleTime,
```

Comment:
Just a nitpick -> I have enabled this globally for all queries https://github.com/flanksource/flanksource-ui/blob/d3f6fb083e72914b46203830f4cc0cabf9ef96ec/src/App.tsx#L52.